database/csv-generator/csv-generator.js

```diff
@@ -0,0 +1,10 @@
+let fakeData = [];
+
+const getFakeData = () => {
```

Comment:
Does this need a parameter? This function is called below with an argument.

Reply:
I was considering sending a value to decide how much data to give, but I decided it didn't matter and forgot to remove the argument from the function call.

Reply:
got it. Looks good.